com.unity.render-pipelines.high-definition/Documentation~/Lighting-Mode-Shadowmask.md

```diff
@@ -7,43 +7,41 @@ The High Definition Render Pipelines (HDRP) supports the Shadowmask Lighting Mod
 To use shadowmasks in HDRP, you must set up your Project to support them. To do this:
 
 1. Enable Distance Shadowmask for every Quality Level in your Unity Project:
-
-2. 1. Open the Project Settings window (menu: Edit > Project Setting) and select the Quality tab.
+   1. Open the Project Settings window (menu: **Edit** > **Project Setting**) and select the Quality tab.
    2. Select a Level (for example, Medium) then, in the Shadows section, set the Shadowmask Mode to Distance Shadowmask.
```

Comment:
UI headers and properties should be bolded. 
So here (double check for accuracy because I haven't seen the UI):
Level 
Shadows 
Distance Shadowmask